libcxx/test/support/asan_testing.h

```diff
@@ -74,17 +74,11 @@ TEST_CONSTEXPR bool is_string_asan_correct(const std::basic_string<ChrT, TraitsT
   if (TEST_IS_CONSTANT_EVALUATED)
```

Comment:
`is_string_short` above can be removed, I think it's not used anymore.

Reply:
Removed.